examples/resources/cloudavenue_edgegateway_security_group/resource.tf

```diff
@@ -3,13 +3,13 @@ resource "cloudavenue_edgegateway_security_group" "example" {
   name            = "example"
   description     = "This is an example security group"
   member_org_network_ids = [
-    cloudavenue_network_routed.example.id
+    cloudavenue_edgegateway_network_routed.example.id
   ]
 }
 
 data "cloudavenue_edgegateways" "example" {}
 
-resource "cloudavenue_network_routed" "example" {
```

Comment:
Remove other resource in the example and in the template use `{{ tffile filename.tf }}` for importing the example